css/style.css

```diff
@@ -298,6 +320,37 @@ body {
   margin: 8px 0;
 }
 
+.modal-container {
+  background-color: rgba(255, 255, 255, 0.16);
+  backdrop-filter: blur(2px);
+  position: fixed;
+  z-index: 5;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  padding: 24px;
+  border: 1px solid #fff;
+  margin: 23px;
+  height: 1128px;
+}

```

Comment:
- [ ] On desktop screen, the modal isn't centered properly, please make sure to address this issue.

![](https://i.imgur.com/lrf9xui.png)

Reply:
THanks for the review change made

Reply:
Thanks for the review change made.